src/instruments/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+rollup -c src/instruments/buildSrc/simulatorBuild.mjs
```

Comment:
Why do we need a separate `.sh` script for this?

Reply:
Forgot to remove it.